aws/route_table.go

```diff
@@ -101,30 +101,32 @@ func (t *RouteTable) GetSrcByVip(vip string) (*Ec2Meta, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), timeOut)
 	defer cancel()
 
-	id, err := t.getSrcByVip(vip)
+	id, state, err := t.getSrcByVip(vip)
 	if err != nil {
 		return nil, err
 	}
 
 	var name string
 	switch {
-	case strings.HasPrefix(id, "i-"):
+	case strings.HasPrefix(id, "i-") && state == "active":
```

Comment:
`RouteStateActive` is in https://github.com/aws/aws-sdk-go/blob/master/service/ec2/api.go.
Would you use it?